src/_P036_FrameOLED.ino

```diff
@@ -378,6 +381,10 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
         bScrollLines = getBitFromUL(PCONFIG_LONG(0), 17);  // Bit 17
         addFormCheckBox(F("Scroll long lines"), F("p036_ScrollLines"), bScrollLines);
 
+        bNoDisplayOnReceivedText = getBitFromUL(PCONFIG_LONG(0), 18);  // Bit 18
+        addFormCheckBox(F("Keep display off receiving text"), F("p036_NoDisplay"), bNoDisplayOnReceivedText);
+        addFormNote(F("Enable this option to not wake-up the display when text is sent from a remote system."));
```

Comment:
I know the default value of this bit is 0 (false), so it would make sense to describe it as a negation.
Better to invert the read and write value and offer the user a choice which is not a negation.

Now a user has to read twice to get the true meaning of this flag.

So use something like this:  "Wake display on receiving text" (or shorter?)
So this means you also have to negate the boolean value for the checkbox and also negate it when storing it. You can leave the current name of that boolean and the rest of the code.